feature-staking-impl/src/main/res/layout/fragment_setup_staking_type.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/redeemContainer"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:background="@color/secondary_screen_background">
+
+    <io.novafoundation.nova.common.view.Toolbar
+        android:id="@+id/setupStakingTypeToolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:dividerVisible="false"
+        app:textRight="@string/common_done"
+        app:titleText="@string/staking_type_title" />
+
+    <io.novafoundation.nova.feature_staking_impl.presentation.view.stakingType.StakingTypeView
+        android:id="@+id/setupStakingTypePoolStaking"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="16dp"
+        android:layout_marginTop="16dp" />
+
+    <io.novafoundation.nova.feature_staking_impl.presentation.view.stakingType.StakingTypeView
+        android:id="@+id/setupStakingTypeDirectStaking"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="16dp"
+        android:layout_marginTop="16dp" />
```

Comment:
Just to make sure: once logic is added, staking types should be determined dynamically based on staking types available for chain